docs/mkdocs/docs/api/arctic_uri.md

```diff
@@ -22,11 +22,45 @@ Available options for S3:
 | access                | S3 access key                                                                                                                                                   |
 | secret                | S3 secret access key                                                                                                                                            |
 | path_prefix           | Path within S3 bucket to use for data storage                                                                                                                   |
-| aws_auth              | If true, authentication to endpoint will be computed via AWS environment vars/config files. If no options are provided `aws_auth` will be assumed to be true.   |
+| aws_auth              | AWS authentication method. If setting is `1` (or `true` for backward compatibility), authentication to endpoint will be computed via [AWS default credential provider chain](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/credproviders.html). If setting is `2`, AWS Security Token Service (STS) will be the authentication method used. If no options are provided AWS authentication will be assumed to be not used. More info is provided below |
+| aws_profile           | Only when `aws_auth` is set to be `2`. AWS profile to be used with AWS Security Token Service (STS). More info is provided below |
```

Comment:
Why is this in the URI? Why don't we just use the `AWS_PROFILE` env var?

Reply:
And it aligns with the boto3 API: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sts/client/assume_role.html
which role is specified in the API, rather than the environment variable

Reply:
OK. The sentence `Only when `aws_auth` is set to be `2`.` is no longer correct